### ISSUE TITLE
Problems installing from pypi, Pastebin.py not found.

### ISSUE THREAD

**Issue by ivanalejandro0** [Morrolan/PastebinAPI#1]
I have this problem installing the package using `pip` and `easy_install`.
Here the logs:

```
(venv:pruebas) ➜ pip install Pastebin
Downloading/unpacking Pastebin
  Downloading Pastebin-1.1.1.tar.gz (55kB): 55kB downloaded
  Running setup.py egg_info for package Pastebin

    file Pastebin.py (for module Pastebin) not found
Requirement already satisfied (use --upgrade to upgrade): setuptools in /home/ivan/.virtualenvs/pruebas/lib/python2.7/site-packages (from Pastebin)
Installing collected packages: Pastebin
  Running setup.py install for Pastebin
    file Pastebin.py (for module Pastebin) not found
    file Pastebin.py (for module Pastebin) not found
    warning: install_lib: 'build/lib.linux-i686-2.7' does not exist -- no Python modules to install


    file Pastebin.py (for module Pastebin) not found
    file Pastebin.py (for module Pastebin) not found
Successfully installed Pastebin
Cleaning up...
```

```
(venv:pruebas) ➜ easy_install Pastebin
Searching for Pastebin
Reading https://pypi.python.org/simple/Pastebin/
Best match: Pastebin 1.1.1
Downloading https://pypi.python.org/packages/source/P/Pastebin/Pastebin-1.1.1.tar.gz#md5=d9374efc32f442e9fb4589d4f5a9b74b
Processing Pastebin-1.1.1.tar.gz
Writing /tmp/easy_install-KfbedV/Pastebin-1.1.1/setup.cfg
Running Pastebin-1.1.1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-KfbedV/Pastebin-1.1.1/egg-dist-tmp-I9DzMO
file Pastebin.py (for module Pastebin) not found
file Pastebin.py (for module Pastebin) not found
file Pastebin.py (for module Pastebin) not found
warning: install_lib: 'build/lib.linux-i686-2.7' does not exist -- no Python modules to install

zip_safe flag not set; analyzing archive contents...
Adding Pastebin 1.1.1 to easy-install.pth file

Installed /home/ivan/.virtualenvs/pruebas/lib/python2.7/site-packages/Pastebin-1.1.1-py2.7.egg
Processing dependencies for Pastebin
Finished processing dependencies for Pastebin
```

Notes:
- I tried installing with `Pastebin` and `pastebin`, no luck.
- I was able to install the app using `python setup.py install` from this repo.

**Comment by comkieffer** [comment]
Hi, 

This issue is still present. Do you have any advice on working around it ? 

	$> pip install pastebin
	Downloading/unpacking pastebin
	  Downloading Pastebin-1.1.1.tar.gz (55kB): 55kB downloaded
	  Running setup.py (path:/home/ ... /build/pastebin/setup.py) egg_info for package pastebin

	    file Pastebin.py (for module Pastebin) not found
	Requirement already satisfied (use --upgrade to upgrade): setuptools in /home/ ... /lib/python3.4/site-packages (from pastebin)
	Installing collected packages: pastebin
	  Running setup.py install for pastebin
	    file Pastebin.py (for module Pastebin) not found
	    file Pastebin.py (for module Pastebin) not found
	    warning: install_lib: 'build/lib' does not exist -- no Python modules to install


	    file Pastebin.py (for module Pastebin) not found
	    file Pastebin.py (for module Pastebin) not found
	Successfully installed pastebin
	Cleaning up...

**Comment by Morrolan** [comment]
I will investigate this over the next few days and get to the bottom of it.  Thanks for bringing it back to my attention.

**Comment by hjwp** [comment]
+1 https://www.pythonanywhere.com/forums/topic/2903/

We're on a linux platform too. As a workaround, reverting to pastebin==1.1.0 seems to work ok.

**Comment by cstromblad** [comment]
Just wanted to add a little comment that in the source package the file is actually called pastebin.py rather than the expected Pastebin.py.

Simply by changing the name of the pastebin.py file into Pastebin.py the manual installation works as expected, i.e. sudo python setup.py install.

Perhaps this might help someone use this package.

**Comment by bartoszek** [comment]
Problem lays in source package hosted on pypi. A `setup.py` file on pypi has `py_modules = ['Pastebin']` which produce the error. Change `'Pastebin'` to `'pastebin'` in `py_modules` fixes the problem.

**Comment by Morrolan** [comment]
Apologies all - this is now resolved in 1.1.2 which is now on pypi.
